test/e2e/specs/specs-playwright/wp-reader__view-spec.ts

```diff
@@ -0,0 +1,56 @@
+import {
+	DataHelper,
+	LoginFlow,
+	setupHooks,
+	ReaderPage,
+	NotificationsComponent,
+	NavbarComponent,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Reader: View and Comment' ), function () {
+	let page: Page;
+	let readerPage: ReaderPage;
+	const comment = DataHelper.randomPhrase();
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log In', async function () {
+		const loginFlow = new LoginFlow( page, 'commentingUser' );
+		await loginFlow.logIn();
+	} );
+
+	it( 'Can see the Reader stream', async function () {
+		readerPage = new ReaderPage( page );
+	} );
+
+	it( 'The latest post is on the expected test site', async function () {
+		const testSiteForNotifications = DataHelper.config.get( 'testSiteForNotifications' );
+		readerPage = new ReaderPage( page );
+		const siteOfLatestPost = await readerPage.siteOfLatestPost();
+		expect( siteOfLatestPost ).toEqual( testSiteForNotifications );
+	} );
+
+	it( 'Can comment on the latest post and see the comment appear', async function () {
```

Comment:
What do you think of breaking from the exact port here and breaking these steps out into two steps? I believe in the newer style guides we've been aiming for smaller steps.